crates/rpc/src/custom_api.rs

```diff
@@ -0,0 +1,7 @@
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+
+#[rpc(server)]
+pub trait CustomApi {
```

Comment:
I think it would be better to have web3.rs and web3_api.rs instead of custom, to follow the same pattern.